.github/workflows/diff-change-to-dist.yaml

```
@@ -42,6 +42,7 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const fs = require('fs').promises
+            const commentOnPr = require('./shared/stats/comment.js').commentOnPr
             const diff = await fs.readFile(
```

I doubt we'll ever run into it but good to have Windows compatibility in mind using `path.join()`

```suggestion
            const { GITHUB_WORKSPACE } = process.env

            const diff = await readFile(join(GITHUB_WORKSPACE, 'dist.diff', 'utf8'))
```

---

Wonder if that's because they've faked `require()` via [wrap-require.ts](https://github.com/actions/github-script/blob/main/src/wrap-require.ts)?

Any better with `const { join } = require('node:path')`?

---

🤦 My bad, the complaint about `path` was from a previous attempt to fix where I was using `path.join`.